src/components/Modal/ModalConfirmation.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Title as DialogTitle, Close as DialogClose } from '@radix-ui/react-dialog';
+import classNames from 'classnames';
+import { Button } from '../Button';
+import { Modal } from '.';
+import { IconButton } from '../IconButton';
+import { IconXClose } from '../Icons';
+
+import styles from './Modal.module.scss';
+
+export interface ModalConfirmationProps {
+  className?: string;
+  title: string;
+  cancelLabel: string;
+  confirmationLabel: string;
+  onCancel: () => void;
+  onConfirm: () => void;
+}
+
+export const ModalConfirmation: React.FC<ModalConfirmationProps> = ({
+  className,
+  children,
+  title,
+  cancelLabel,
+  confirmationLabel,
+  onCancel,
+  onConfirm,
+  ...rest
+}) => {
+  return (
+    <Modal className={classNames(styles.WiderContent, className)} {...rest}>
+      <div className={styles.Head} data-testid="modal-head">
+        <DialogTitle className={styles.Title}>{title}</DialogTitle>
+
+        <DialogClose asChild className={styles.CloseIcon}>
+          <IconButton Icon={IconXClose} onClick={onCancel} />
```

Comment:
Hi @colbr , I tried to reuse the icon that we have in the storybook with white color, but I'm not sure why it's now black instead of why. do you know what is the issue ?

Reply:
Took a quick look. I think it's the default browser style on Button components. We probably should have a global reset style or something so that buttons inherit the color.

Reply:
![image](https://user-images.githubusercontent.com/43770/223746458-79ad53c5-2316-4949-80b5-312c9f294b17.png)

Reply:
I forgot to add which button I wanted to reuse from storybook: https://zui-storybook.netlify.app/?path=/story/inputs-iconbutton--close-icon.

Reply:
> ![image](https://user-images.githubusercontent.com/43770/223746458-79ad53c5-2316-4949-80b5-312c9f294b17.png)

you are right, I've removed the color.